### DISCUSSION TITLE
Message: Connection out

### DISCUSSION THREAD

**Discussion by darkkness-creator** [lllyasviel/Fooocus#3700] [General]
Hi, Sometimes I get this message when I'm working with Fooocus in Inpaint/Outpaint. I have to reboot to get it working properly again. Can someone tell me why I get this message. Thanks.

Computer: Ryzen 7 7735HS with 32GB RAM and NVIDIA RTX 4060 Laptop 8GB.

**Comment by darkkness-creator** [comment]
Anyone?.

**Comment by ghost** [comment]
Look for the real error message in the terminal/cmd window.
